docs/source/getting_started/lightly_at_a_glance.rst

```diff
@@ -3,14 +3,14 @@
 Self-supervised learning
 ========================
 
-Lightly is a computer vision framework for training deep learning models using self-supervised learning.
+Lightly SSL is a computer vision framework for training deep learning models using self-supervised learning.
 The framework can be used for a wide range of useful applications such as finding the nearest 
 neighbors, similarity search, transfer learning, or data analytics.
 
 
-How Lightly Works
+How Lightly SSL Works
 -----------------
-The flexible design of Lightly makes it easy to integrate in your Python code. Lightly is built
+The flexible design of Lightly SSL makes it easy to integrate in your Python code. Lightly SSL is built
```

Comment:
Minor: Lengthen the dashes on the line above to match the header (I'm not sure whether sphinx understands the formatting otherwise, maybe it does).

Reply:
just checked; needs to be at least as long as the text... very weird "spec" for sure